firmware/controllers/algo/engine_configuration.cpp

```diff
@@ -325,11 +325,26 @@ void prepareVoidConfiguration(engine_configuration_s *engineConfiguration) {
 
 	engineConfiguration->sdCardSpiDevice = SPI_NONE;
 	boardConfiguration->sdCardCsPin = GPIO_UNASSIGNED;
-	engineConfiguration->accelerometerSpiDevice = SPI_DEVICE_1;
```

Comment:
This should not be in "void" configuration. We move it to setDefaultBoardConfiguration()